src/Flash/__tests__/__snapshots__/Flash.test.tsx.snap

```diff
@@ -3,13 +3,13 @@
 exports[`Flash renders consistently 1`] = `
 .c0 {
   position: relative;
-  color: #1F2328;
+  color: #24292f;
```

Comment:
I'm confused by all of these snapshot updates with different token values. It doesn't look like you changed anything that would cause this. Am I missing something? 🤔

Reply:
I am too. Looking into it.